static/app/views/replays/detail/issueList.tsx

```diff
@@ -68,28 +73,38 @@ function IssueList(props: Props) {
         issues: [],
       });
     }
-  }, [api, organization.slug, props.replayId, props.projectId]);
+  }, [api, organization.slug, replayId, projectId]);
 
   useEffect(() => {
     fetchIssueData();
   }, [fetchIssueData]);
 
+  const counts = useReplaysCount({
+    groupIds: state.issues.map(issue => issue.id),
+    organization,
+    project,
+  });
```

Comment:
There is only one project in this context, so we can skip `<ReplayCountProvider>` that all the other instances use and call the hook directly. Also this and `GroupListBody` are the only functional components that could use a hook.